docs/source/daml/intro/daml/daml-intro-5/daml/Restrictions.daml

```diff
@@ -54,13 +54,13 @@ template SimpleIou
           dow = dayOfWeek today
           timeofday = now `subTime` time today 0 0 0
           hrs = convertRelTimeToMicroseconds timeofday / 3600000000
-        assertMsg
-          ("Cannot redeem outside business hours. Current time: " <> show timeofday)
-          (hrs >= 8 && hrs <= 18)
-        case dow of
-          Saturday -> abort "Cannot redeem on a Saturday."
-          Sunday -> abort "Cannot redeem on a Sunday."
-          _ -> return ()
+        if (hrs < 8 || hrs > 18) then
```

Comment:
```suggestion
        if (hrs < 8 || hrs > 18) then
          abort "Cannot redeem outside business hours. Current time: " <> show timeofday
        else case dow of
          Saturday -> abort "Cannot redeem on a Saturday."
          Sunday -> abort "Cannot redeem on a Sunday."
          _ -> return ()
```

Reply:
I think the case statement is a good idea, just the use of `assertMsg` adds an unnecessary concept at this point in the lessons.

Reply:
The `$` is necessary given the precedence of `<>`. Also while I haven't included the correction, in Sydney we never use `return` preferring `pure` instead to avoid potential conflation of the function here with its use as a keyword in most other languages.